public/index.html

```diff
@@ -50,6 +51,11 @@
       <script src="https://developers.redhat.com/modules/contrib/red_hat_universal_navigation/dist/%40chapeaux/cpx-user/cpx-user.js?rrg73h"></script>
       <script src="https://developers.redhat.com/modules/contrib/red_hat_universal_navigation/js/rh-account-dropdown.js?rrg73h"></script>
       <script src="https://developers.redhat.com/modules/contrib/red_hat_universal_navigation/dist/%40cpelements/rh-site-switcher/dist/site-switcher.umd.min.js?rrg73h"></script>
+      <script type="text/javascript">
+        if (("undefined" !== typeof _satellite) && ("function" === typeof _satellite.pageBottom)) {
+            _satellite.pageBottom();
+        }
+      </script>
```

Comment:
shouldn't this be in the body? i.e. outside the div?

> Add the following markup as the very last element in your HTML <body> element:

Reply:
updated! :+1: